### PR TITLE
Adds first example

### PR DESCRIPTION
**Pull request by bbotella**

### CODE REVIEW

File: examples/merge_data/main.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+from exporters.filters.bloom_filter import DuplicatesBloomFilter
+from exporters.meta import ExportMeta
+from exporters.readers import RandomReader
+from exporters.writers import ConsoleWriter
+logging.basicConfig()
+
+
+if __name__ == '__main__':
+
+    # Ozzy modules need a shared meta object, to store shared information
```
Comment (bbotella) [inline examples/merge_data/main.py]: We should ditch this name

File: docs/modules/exporters.bypasses.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
+exporters.bypasses package
+==========================
```
Comment (eliasdorneles) [inline docs/modules/exporters.bypasses.rst]: was this bypasses.rst file autogenerated?

Comment (bbotella) [inline docs/modules/exporters.bypasses.rst]: Right. Let me remove it.

**Comment by eliasdorneles** [comment]
I think we should remove the `__init__.py` from the examples dir -- it's not meant to be a Python module, right?

**Pull request merged**